_inc/client/writing/theme-enhancements.jsx

```diff
@@ -198,6 +200,27 @@ class ThemeEnhancements extends React.Component {
 						) }
 					</SettingsGroup>
 				) }
+				{ foundCustomCSS && (
+					<SettingsGroup
+						module="custom-css"
```

Comment:
Module here should be the object of the module. Currently React is complaining about the prop type.

Reply:
Fixed